sdk/communication/communication-chat/src/models/mappers.ts

```diff
@@ -48,14 +60,17 @@ export const mapToChatContentSdkModel = (
 };
 
 /**
+ * @internal
  * Mapping chat message REST model to chat message SDK model
  */
 export const mapToChatMessageSdkModel = (chatMessage: RestModel.ChatMessage): ChatMessage => {
-  const { content, senderId, ...otherChatMessage } = chatMessage;
+  const { content, senderCommunicationIdentifier, ...otherChatMessage } = chatMessage;
   const contentSdkModel = content ? mapToChatContentSdkModel(content) : undefined;
-  if (senderId) {
+  if (senderCommunicationIdentifier) {
     return {
-      sender: { communicationUserId: senderId! },
+      sender: deserializeCommunicationIdentifier(
+        senderCommunicationIdentifier as SerializedCommunicationIdentifier
```

Comment:
Why is this type assertion necessary, it shouldn't? If TS can't infer that it matches the interface it's a sign that something has gone wrong.

Reply:
**In common model:**
 `cloud?: SerializedCommunicationCloudEnvironment;`
 `export declare type SerializedCommunicationCloudEnvironment = "public" | "dod" | "gcch";`
In generated model
`export type CommunicationCloudEnvironmentModel = string;`